hera_mc/mc_session.py

```diff
@@ -1040,6 +1047,151 @@ def get_roach_temperature(self, starttime, stoptime=None, roach=None):
                                  stoptime=stoptime, filter_column='roach',
                                  filter_value=roach)
 
+    def add_node_sensor(self, time, node, top_sensor_temp, middle_sensor_temp,
+                        bottom_sensor_temp, humidity_sensor_temp, humidity):
+        """
+        Add new node sensor data to the M&C database.
+
+        Parameters:
+        ------------
+        time: astropy time object
+            astropy time object based on a timestamp reported by node
+        node: integer
+            node number (integer running from 1 to 30)
+        top_sensor_temp: float
+            temperature of top sensor reported by node in Celcius
```

Comment:
Not a huge deal, but spelling: "celsius".

Reply:
Thanks, that's embarrassing.